### PR TITLE
Adding electron_rest_mass unit in physics.units

### LINKED ISSUE #24462: Missing electron rest mass unit in physics.units
The electron rest mass (symbol: me) is the mass of a stationary electron, also known as the invariant mass of the electron. It is one of the fundamental constants of physics. It has a value of about 9.109×10−31 kilograms
It is not in `physics.units` should we add it.

### PR DESCRIPTION
**Pull request by 1e9abhi1e10**
#### References to other Issues or PRs

Fixes #24462 

#### Brief description of what is fixed or changed
The electron rest mass (symbol: me) is the mass of a stationary electron, also known as the invariant mass of the electron. It is one of the fundamental constants of physics.

#### Release Notes

* physics.units
  * Added electron rest mass unit

### CODE REVIEW

File: sympy/physics/units/tests/test_quantities.py [hunk 1]
```diff
@@ -304,11 +307,11 @@ def test_find_unit():
...
         'centiliters', 'milliliters', 'planck_volume']
     assert find_unit('voltage') == ['V', 'v', 'volt', 'volts', 'planck_voltage']
-    assert find_unit(grams) == ['g', 't', 'Da', 'kg', 'mg', 'ug', 'amu', 'mmu', 'amus',
-                                'gram', 'mmus', 'grams', 'pound', 'tonne', 'dalton',
-                                'pounds', 'kilogram', 'kilograms', 'microgram', 'milligram',
-                                'metric_ton', 'micrograms', 'milligrams', 'planck_mass',
-                                'milli_mass_unit', 'atomic_mass_unit', 'atomic_mass_constant']
+    assert find_unit(grams) == ['g', 't', 'Da', 'kg', 'me', 'mg', 'ug', 'amu', 'mmu', 'amus',
+                                'gram', 'mmus', 'grams', 'pound', 'tonne', 'dalton', 'pounds',
+                                'kilogram', 'kilograms','microgram', 'milligram', 'metric_ton',
```
Comment (ThePauliPrinciple) [inline sympy/physics/units/tests/test_quantities.py:312]: misses a space between `kilograms` and `microgram`

**Comment by ThePauliPrinciple** [comment]
Looks good to me.

Not directly related to this PR, but it seems inconvenient to add more units. The same (as well as very similar) code needs to be added in different files.

**Comment by 1e9abhi1e10** [comment]
> Looks good to me.
> 
> Not directly related to this PR, but it seems inconvenient to add more units. The same (as well as very similar) code needs to be added in different files.


I also added an angstrom unit via  #24425, what can I do with that 
Should I close that one ?

**Comment by ThePauliPrinciple** [comment]
Maybe it's a good idea to investigate which "common" units SymPy is missing and add them in the same PR. 

It would also be helpful if you can squash commits that are just typos. If you're not familiar with squashing in git, then make sure you make a hard backup (you're overriding the version control, so you can't rely on git to back up what you have done).

**Comment by sylee957** [comment]
There should be a release note for this

**Comment by 1e9abhi1e10** [comment]
> There should be a release note for this

Done!

**Pull request merged**